main.go

```diff
@@ -161,11 +164,34 @@ func walk(ch chan<- *file, start string) {
 		if fi.IsDir() {
 			return nil
 		}
+		if excludePath.Contains(path) {
+			log.Printf("%s skipped", path)
+			return nil
+		}
 		ch <- &file{path, fi.Mode()}
 		return nil
 	})
 }
 
+type ExcludePath struct {
+	paths []string
+}
+
+func NewExcludePaths(arg string) *ExcludePath {
+	return &ExcludePath{
+		paths: strings.Split(arg, ","),
+	}
+}
+
+func (e *ExcludePath) Contains(path string) bool {
+	for _, p := range e.paths {
+		if p == path {
```

Comment:
Will this work if someone specifies a relative path, e.g. /foo/bar, ./foo, ../foo/bar?
Perhaps you should canonicalize both strings to a full path to ensure you catch all those cases.